draft-ietf-ppm-dap.md

```diff
@@ -725,11 +725,13 @@ struct {
 struct {
   TaskID task_id;
   ReportMetadata metadata;
+  opaque public_share<0..2^32-1>;
```

Comment:
AFAICT, `public_share` always accompanies `ReportMetadata`, so it'd be nice to tuck the former into the latter, because then the public share would get included in message AAD and appear in all the places it needs to "for free". My only reservation is that calling the public share "metadata" feels like a misnomer.

Reply:
Yes, you're right. We could rename to "header" maybe, but @chris-wood rejected this name in an earlier PR. Maybe it makes sense to rename if we include public share?